pre-commit/tests/data/py3-app/expected_app_json.out

```diff
@@ -2,73 +2,17 @@
     "python_version": "3",
     "pip_dependencies": {
         "wheel": [
-            {
-                "module": "beautifulsoup4",
-                "input_file": "wheels/py3/beautifulsoup4-4.9.1-py3-none-any.whl"
-            },
-            {
-                "module": "certifi",
-                "input_file": "wheels/py3/certifi-2024.2.2-py3-none-any.whl"
-            },
-            {
-                "module": "charset_normalizer",
-                "input_file": "wheels/py3/charset_normalizer-2.0.12-py3-none-any.whl"
-            },
-            {
-                "module": "idna",
-                "input_file": "wheels/py3/idna-3.6-py3-none-any.whl"
-            },
-            {
-                "module": "requests",
-                "input_file": "wheels/shared/requests-2.26.0-py2.py3-none-any.whl"
-            },
             {
                 "module": "simplejson",
```

Comment:
Not really for this PR, but there's really no reason to use `simplejson` at this point. It's long since been ported into the python standard library as the canonical implementation of `json`. It'd be nice to update apps to just use the bulitin `json` module directly

Reply:
Yeah. I'm not sure why the example app in the tests is pulling it in.

Certainly I hope no production app is using it, but they probably are 😐. Maybe we need a second list, that fails the pre-commit or pipeline if [x] dependency is included 🙃.